game.js

```diff
@@ -0,0 +1,80 @@
+// get choice of computer in random
+function computerChoice() {
+  const options = ["Rock", "Paper", "Scissors"];
+  const choice = options[Math.floor(Math.random() * options.length)];
```

Comment:
No need to assign it in a variable if you are returning it immediatly.

return options[Math.floor(Math.random() * options.length)];